internal/internal_workflow_client.go

```diff
@@ -149,98 +150,6 @@ type (
 	}
 )
 
-// StartWorkflow starts a workflow execution
```

Comment:
While large swaths of code like this look like they were deleted, they were just moved to an interceptor implementation so these calls could be intercepted.